restapi/src/main/java/io/stargate/web/docsapi/resources/ReactiveDocumentResourceV2.java

```diff
@@ -319,6 +346,12 @@ public void putDocPath(
           @NotNull(message = "payload not provided")
           @NotBlank(message = "payload must not be empty")
           String payload,
+      @ApiParam(
+              value =
+                  "Include this to put a time-to-live on the document in the data store; use 'auto' on a sub-document to make the TTL match that of the parent document.",
+              required = false)
+          @QueryParam("ttl")
+          String ttl,
```

Comment:
let's have a different name for the param and make it a boolean, see other comment